### ISSUE TITLE
Design the main character

### ISSUE THREAD

**Issue by erik90mx** [KOBUGE-Games/ringed#47]
Well right now I finish this issue: https://github.com/KOBUGE-Games/ringed/issues/45

But for a better feeling of the recoil shot, it will need arm animation of the character, right now it is just one part, but we will need more animations in order to transmit the player a more dynamic character.

I will do the 3D model and rigg of the character (to get the animation sprites fast from the reders), but the first step that we need, is an objective character design or concept art.

Please propose in this issue things about the charater (references, ideas, early draws and concepts), and if you can make a concept art it will be very welcome and helpful.

**Comment by mamarilmanson** [comment]
I just need some requirements:

Sex:
Race/species:
Ethnicity:
Age:
How old they appear:
Occupation: 
A little back story:

Just need a more professional approach... :D

**Comment by mamarilmanson** [comment]
and then when the 3d part is ready... we'll convert it to 3d... XD

**Comment by erik90mx** [comment]
Well I comment the issue with a writer that I know, and she give me this like a first try
**Story 1 Character Profile**
- **Sex:** Masculine
- **Race/species:** Human
- **Ethnicity:** American
- **Age:** 43
- **How old they appear:** 51
- **Occupation:** Night Security Guard
- **A Little Back Story:** He was in the army and participated in the war, but after a leg injury had to retire, when he returned home to live with his wife, she left him. He has worked 15 years in the company that the government recommends. He is a quiet person, a bit boring, he does not care about the life and is misaligned.
  He never had children and keeps unstable relationships with different types of women.

"Heroes do not need to be perfect to be heroes"

**Comment by mamarilmanson** [comment]
Very big company description... thanks... :+1:

**Comment by erik90mx** [comment]
I will put details of that in the history issue @mamarilmanson  :)   , I'm working that with the writer right now, maybe I update it tomorrow.

**Comment by ObaniGemini** [comment]
Wasn't it a women ?

>  "Heroes do not need to be perfect heroes"

Action film/games heroes do not need to be the all same heroes :/

**Comment by erik90mx** [comment]
@ObaniGemini  yep that's the point ;)

@mamarilmanson  here is the story that I discuss with the writer in the phone, we can modify it for sure:  https://github.com/KOBUGE-Games/ringed/issues/19

It is just to help us to get a path in the develop.

**Comment by mamarilmanson** [comment]
:+1:

**Comment by erik90mx** [comment]
Right now there is other character profile in this issue: https://github.com/KOBUGE-Games/ringed/issues/19

**Comment by mamarilmanson** [comment]
Based on the character profile above I came with this... although a work in progress please comment ahead to know if we are going the right direction... thanks... :D

![char_des_wip](https://cloud.githubusercontent.com/assets/3820189/8638144/653bf97c-28e3-11e5-8163-d0723b891828.png)

**Comment by erik90mx** [comment]
@mamarilmanson I really really like the approach you get of this first character profile, his face attitude is what catch my attention more, it reflects the kind of person that the profile transmit me (nice touch with the hair too), I like his body constitution (that was an issue when I was talking with the writer) because he will be not in shape but will be not fat at all.

Dont forget this thing here: http://www.webdelsol.com/DIAGRAM/2_4/pr24_tech.jpg

It will be the main weapon at the begining in that story.

The kind of boots or pants in them, reminds me how explorers use them, maybe a more straight pant end?
http://www.rastro.com/images/5041749_20150213112404154de090439e5e.jpg

Like in that image, I always think in a security guard with a tie:
http://www.sesp.com.mx/images/guard.jpg
what do you think about it? is very cliche or something?

I almost can imagine him drinking :)

Good work in that concept @mamarilmanson

**Comment by bojidar-bg** [comment]
@mamarilmanson :+1:

**Comment by mamarilmanson** [comment]
@erik90mx ~ my reasoning about removing the necktie is it kind of chokes the player and it's inconvenient during war... and about the boots, he is a war veteran so he knows what to wear when to wear... 

no harm intended... :D

**Comment by erik90mx** [comment]
@mamarilmanson I take in consideration the story time line, in where the character is currently working his shift in the company like a normal security guard (like normal companies this one maybe has a standard dress code and assigned uniform), and at the time the things happened all the building was lock down, to take that clothes in you concept, maybe after a while exploring the Scientific installation he will found some army boots and stuff that can be helpful (it is good that the company has Government contracts) that can give us a possible explanation, but is hard to think in that costume at the moment the game begin.

I will copy here the other 2 profiles

**_Story 2 Character Profile:**_
- **Sex:** Men
- **Race/species:** Human
- **Ethnicity:** Asian
- **Age:** 35 years
- **How old they appear:** 1,74 meters y 68 kilograms; Thin; white skin; black, short and straight air; Asian and black eyes. 
- **Occupation:** Lawyer
- **City:** Tokyo
- **A little back story:** He is the only son, his parents are entrepreneurs and all his life try to be the best, no matter how difficult the situation is. Be perfect was the daily normal routine, so after long sleepless nights, achieving his goals and lead a life full of sacrifices, he managed to enter the lawyer’s firm that he wanted. He married at 30 and has 1 year old son. He is a very serious guy, with obtuse mind, patient and astute.

**_Story 3 Character Profile:**_
- **Sex:** Male
- **Race/species:** Alien/Dog
- **Ethnicity:** Unknow 
- **Age:** 42 human years
- **How old they appear:** It is an alien race that looks like dogs, but they can speak, walk in two feet and do almost everything that a human can do. His face looks like an adorable puppy, although his true personality is aggressive and feral. He has a scar in the left eye. He is 64 centimeters tall and has a weight according to his body. When is dressing himself normally wear very formal clothes, a weapon in his waist and a pure in his snout.
- **Occupation:** Intergalactic Police
- **A little back story:** He is divorced, has four puppies that see in holydays. Was working for the agency almost 15 years, but because of his horrible character, he is not very popular between his teammates. The agency forced him to take a vacation, because a trouble that he had after knocks out a teammate to mock about his thick and sensual voice.

**Comment by mamarilmanson** [comment]
I like the dog race... :D

**Comment by erik90mx** [comment]
@mamarilmanson yep that's the funny story :)  , but with the survey that @bojidar-bg made I think that the winner will be the Story No. 2

**Comment by erik90mx** [comment]
Here a fast sketch about the second character, maybe he will need have a more square head and a tuxedo.
@mamarilmanson  feel free to change all this concept, I will just throwing ideas.
![character2](https://cloud.githubusercontent.com/assets/3485891/8823047/7c8a5634-302f-11e5-986a-b16d69425ed3.jpeg)

**Comment by bojidar-bg** [comment]
@erik90mx :+1: not bad... :smiley: He looks quite distressed, though.. Maybe he should be happy because he got some days off?

@erik90mx ~ BTW, are you sure it is a fast sketch?

**Comment by erik90mx** [comment]
@bojidar-bg yep he need to look more stressed :) and now I think that older too, and yes it is a fast sketch before sleep, it doesn't even have a stylized shape or precise line art.

**Comment by bojidar-bg** [comment]
@erik90mx ~ Just a side note - if you change the head a bit, he will look just like me :stuck_out_tongue:. (well, with the exception of the clothes...)

**Comment by mamarilmanson** [comment]
I have some sketches regarding the original design I made... I'll post when I have time... :)

**Comment by mamarilmanson** [comment]
A work in progress... 

![sitting](https://cloud.githubusercontent.com/assets/3820189/8892204/71b8da84-337f-11e5-9a28-c4376575ae9d.png)

**Comment by bojidar-bg** [comment]
@mamarilmanson not bad, though he looks a bit too old IMO.

**Comment by mamarilmanson** [comment]
that was from the war veteran, stressed and a bit isolated personality from before... :)

**Comment by erik90mx** [comment]
@mamarilmanson  I asked to @bojidar-bg  about the winner story, and it was the second one. I like your concepts, and this one is a really good one  WIP :)

but in order to focus the develop and the story flow, we need all you artist powers giving love to the second story.

lets go for the alpha, wajuu!

**Comment by mamarilmanson** [comment]
2nd story it is... ok... :+1:

**Comment by bojidar-bg** [comment]
FYI, first story scored 1 vote, second scored 6 votes and the third scored 1 point. (first/second/last - 1/6/1)

**Comment by mamarilmanson** [comment]
oh ok hehe

I actually liked the 3rd one despite not voting... I was expecting some humor... :)

**Comment by erik90mx** [comment]
@mamarilmanson  I feel you T_T , I voted for the 3rd one and the 2nd one, but well we need to focus the power to one story.

**Comment by mamarilmanson** [comment]
yeah... T_T so sad...

**Comment by erik90mx** [comment]
@mamarilmanson the writer told me that she would like to put the main character from the 3rd story in the second one (she is thinking about that), and give some kind of humor to the story, according to its mood.
Her computer wasn't working, but now it works, she want to know what precise ideas do we have for the second story, she needs our opinion and ideas to work in something, and give us something to enjoy.
@KOBUGE-Games/owners

**Comment by mamarilmanson** [comment]
ok... I've just read it so I'll think about it...

**Comment by mamarilmanson** [comment]
Some ideas lurking in my mind...
![img_20150806_174213](https://cloud.githubusercontent.com/assets/3820189/9110801/35d3a786-3c73-11e5-9a7d-ce832c60fec5.jpg)

I don't know why it's sideways... but on my local computer it's facing the right way...

**Comment by bojidar-bg** [comment]
@mamarilmanson e.g.? :smiley:

**Comment by erik90mx** [comment]
@mamarilmanson  I don't get it :v

**Comment by erik90mx** [comment]
@mamarilmanson now I see it, it is for the enemies?
PD: I like the way they are waiting in the boxes, very relaxed and expecting the player :)

**Comment by akien-mga** [comment]
> I don't know why it's sideways... but on my local computer it's facing the right way...

That's because you took the photo sideways: the camera orientation is saved in the metadata of the JPG file, so you picture viewer processes this data and turns the picture at runtime. But the JPG image in itself is still rotated, so it appears as such if you put it on the Internet or open it in an image editor for example.

**Comment by bojidar-bg** [comment]
@mamarilmanson Not bad, the only _(little)_ problem is that I have no idea how to implement it :smile:.... But that's... another story :smile_cat:

**Comment by mamarilmanson** [comment]
@bojidar-bg ~ uhm... implement what?

note: those blue lines are dead spiders... and our character is relaxing at one of the crates... :)

these are just mixed up illustrations... sorry if I didn't divide it...

**Comment by bojidar-bg** [comment]
@mamarilmanson :o: I thought it was some enemy resting on the crate :smile: LOL. NVM my last comment...

**Comment by akien-mga** [comment]
Wow, I see there's many interesting ideas and concept art here that I missed, I need to give it a closer look :D

**Comment by mamarilmanson** [comment]
Please forgive the image orientation and also the quality of my camera phone:

Some ideas I have in mind... :)

![img_20150829_160630](https://cloud.githubusercontent.com/assets/3820189/9561921/c629f1f6-4e8d-11e5-9cba-7fdde3329b8c.jpg)

**Comment by erik90mx** [comment]
@mamarilmanson try using this app for take doc or draws pics https://play.google.com/store/apps/details?id=com.intsig.camscanner&hl=es_419
It is very useful, and I always use it to upload images of drawings :)

**Comment by mamarilmanson** [comment]
@erik90mx ~ The main prob is my camera phone's resolution...

anyway... any thoughts??? :)

**Comment by ObaniGemini** [comment]
Google play is evil
F-Droid might be better

**Comment by est31** [comment]
@ObaniGemini precisely my thought... We are too much free software crowd 😋

**Comment by erik90mx** [comment]
@mamarilmanson maybe its the jaw, but it seems a bit from american ethnicity.

**Comment by mamarilmanson** [comment]
Somehow I made my graphics tablet work... here's some sketch I made... let's start development again! :D

please comment on the direction we are about to take regarding the character... :)

![concept art](https://cloud.githubusercontent.com/assets/3820189/9843653/fbf999d6-5aee-11e5-9653-3a203d8348db.jpeg)

**Comment by mamarilmanson** [comment]
Hello??? anyone there??? :)

**Comment by bojidar-bg** [comment]
@mamarilmanson ~ They are good, so we have not much to comment :stuck_out_tongue_winking_eye:

**Comment by mamarilmanson** [comment]
oh... ok...

@bojidar-bg ~ which one though would be the choice? or atleast in the nearest direction? :)

**Comment by erik90mx** [comment]
@mamarilmanson Sorry I was very busy in these weeks :S
I like the chin and overall head of the middle one, but with the mouth and hair of the third one (from left to right), I think that the middle one, needs to have lower glasses (right now it gives the impression of a more long face), their necks feels a bit strong considering that he will be not an athletic man.

Right now the middle one remembers me a lot to this character: http://screenrant.com/wp-content/uploads/Sean-Harris-in-Mission-Impossible-5.jpg

Greetings and good concepts :D

**Comment by mamarilmanson** [comment]
Yes... thanks... will do... since I got my main pc back from the dead... :)

**Comment by est31** [comment]
The left one reminds me of Arnold Schwarzenegger.
